### ISSUE TITLE
Errors in Saving model_weights with PlaidML backend: save_weights() got an unexpected keyword argument 'save_format'

### ISSUE THREAD

**Issue by rajafan** [rstudio/keras#994]
I am currently experiencing an error when trying to save Keras model weights in R using PlaidML. The model saving function works correctly when using the conventional tensorflow backend, but failed when I change the backend to plaidml.
The error is listed as below, showing that save_weights() got an unexpected keyword argument "save_format".
Uploading Screen Shot 2020-02-26 at 1.05.24 PM.png…

A minimal for error reproduction:
`library(keras)

use_backend(backend = 'plaidml')

batch_size <- 128
num_classes <- 10
epochs <- 12

img_rows <- 28
img_cols <- 28

mnist <- dataset_mnist()
x_train <- mnist$train$x
y_train <- mnist$train$y
x_test <- mnist$test$x
y_test <- mnist$test$y

x_train <- array_reshape(x_train, c(nrow(x_train), img_rows, img_cols, 1))
x_test <- array_reshape(x_test, c(nrow(x_test), img_rows, img_cols, 1))
input_shape <- c(img_rows, img_cols, 1)

x_train <- x_train / 255
x_test <- x_test / 255

y_train <- to_categorical(y_train, num_classes)
y_test <- to_categorical(y_test, num_classes)

model <- keras_model_sequential() %>%
  layer_conv_2d(filters = 32, kernel_size = c(3,3), activation = 'relu') %>% 
  layer_max_pooling_2d(pool_size = c(2, 2)) %>% 
  layer_flatten() %>% 
  layer_dense(units = 128, activation = 'relu') %>% 
  layer_dense(units = 10, activation = 'softmax')

model %>% compile(
  loss = loss_categorical_crossentropy,
  optimizer = optimizer_adadelta(),
  metrics = c('accuracy')
)

model %>% fit(
  x_train, y_train,
  batch_size = batch_size,
  epochs = 1,
  validation_split = 0.2
)

save_model_weights_hdf5(object = model, filepath = '/Users/fan/Desktop/a.h5')
`

Does it have anything to do with the package versions? I am using Python 3.6, Keras 2.2.4 and PlaidML 0.7.0. Looking forward to any help, thanks a lot!

**Comment by dfalbel** [comment]
For a quickfix you could try  running `model$save_weights(filepath='path/to/weight.hdf5')`

**Comment by dkgaraujo** [comment]
As a contribution, I would like to say that the solution posted above by dfalbel worked for me (I was having the same issue as the original poster). But for information, I also got the following warning:

```/Users/douglasaraujo/Library/r-miniconda/envs/r-plaidml-keras/lib/python3.6/site-packages/keras/engine/sequential.py:110: UserWarning: `Sequential.model` is deprecated. `Sequential` is a subclass of `Model`, you can just use your `Sequential` instance directly.
  warnings.warn('`Sequential.model` is deprecated. ' ```
